spring-integration-core/src/main/java/org/springframework/integration/channel/AbstractMessageChannel.java

```diff
@@ -175,12 +174,14 @@ public void setLoggingEnabled(boolean loggingEnabled) {
 		this.managementOverrides.loggingConfigured = true;
 	}
 
-	protected AbstractMessageChannelMetrics getMetrics() {
+	protected org.springframework.integration.support.management.AbstractMessageChannelMetrics getMetrics() {
```

Comment:
Shouldn't methods like this be deprecated as well?
Kinda what is going to happen to them when we remove classes for their signature?..